css/properties/-webkit-line-grid.json

```diff
@@ -20,7 +21,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "≤13.1"
+              "version_added": "6"
```

Comment:
Added in Safari 6 (https://github.com/caugner/css-properties/commit/3829e0a128a03f20baf7931a0a6d47c5043fedd6).